app/rdf/query-hierarchies.ts

```diff
@@ -26,6 +26,16 @@ const hasValueAndLabel = (
   return o.label && o.value ? true : false;
```

Comment:
I know it was not a part of this PR, but wouldn't just o.label && o.value work the same? 😅

Reply:
Actually only the o.label && o.value would return a string instead of a boolean ;) I replaced by !!(o.value && o.label). Thanks :)